hangman/game.py

```diff
@@ -1,13 +1,101 @@
 from .exceptions import *
+import random
 
 
 class GuessAttempt(object):
-    pass
-
+    new_word = ""
+    def __init__(self, guess, hit=False, miss=False):
+        self.hit = hit
+        self.miss = miss
+        self.guess = guess
+        if self.hit == True and self.miss== True:
```

Comment:
Remember that it's preferred to say `is True` instead of `== True`. Or just, `if self.hit and self.miss:`.